Library/Homebrew/cmd/tests.rb

```diff
@@ -1,3 +1,7 @@
+#:  * `tests` [`-v`] [`--coverage`] [`--generic`] [`--no-compat`] [`--only=`<test_script/test_method>] [`--seed` <seed>] [`--trace`]:
+#:    Run Homebrew's unit and integration tests.
+#: @hide_from_man_page
```

Comment:
If you don't have very strong feelings about the placement of this, I'd prefer that to be the very first comment line instead of being tucked underneath all the documentation (possibly extensive once it fully explains all options) and thus where it's more easily overlooked.